src/functionalTests/java/uk/gov/hmcts/reform/finrem/functional/bulkscan/CcdBulkScanIntegrationTest.java

```diff
@@ -59,6 +59,7 @@ public void givenOcrPayload_whenTransformedPayloadUploadedToCcd_thenCaseIsCreate
 
         try {
             UserDetails userDetails = idamUtils.createCaseworkerUser();
+            log.info("case data = {}, user details = {}", caseData, userDetails);
```

Comment:
this test is broken and I tried to get more details.

Reply:
can be removed now?

Reply:
https://github.com/hmcts/finrem-case-orchestration-service/pull/293